lib/rubocop/cop/style/variable_interpolation.rb

```diff
@@ -5,7 +5,6 @@ module Cop
     module Style
       # This cop checks for variable interpolation (like "#@ivar").
       class VariableInterpolation < Cop
-        # rubocop:disable Lint/InterpolationCheck
```

Comment:
There's a matching `rubocop:enable` directive a few lines down that needs ✂️. 🙂

Reply:
Thanks for the review. I removed a matching `rubocop:enable` and rebased.